lib/friendly_shipping/services/ship_engine.rb

```diff
@@ -22,8 +22,11 @@ def initialize(token:, test: true, client: self.class::Client)
       end
 
       def carriers
-        path = API_BASE + API_PATHS[:carriers]
-        client.get(path, request_headers).fmap do |response|
+        request = FriendlyShipping::Request.new(
+          url: API_BASE + API_PATHS[:carriers],
+          headers: request_headers
+        )
+        client.get(request).fmap do |response|
           ParseCarrierResponse.new(response: response).call
```

Comment:
For consistency, shouldn't this be `ParseCarrierResponse.(response: response)`?

Reply:
Yes, both of these observations are correct. Will amend